### ISSUE TITLE
CC and other licences

### ISSUE THREAD

**Issue by arockenberger** [uio-carpentry/studyGroup#21]
I propose we have a session on CC licences and other licences used in the digital sphere. Especially when it comes to publishing stuff on GitHub and on data repositories (incl. article platforms like Zenodo). Can be done next year!

**Comment by annefou** [comment]
Excellent idea!

The CodeRefinery project (https://coderefinery.org/) has a lesson on it (http://cicero.xyz/v2/remark/github/coderefinery/software-licensing/master/talk.md/#1); 

I am planning to discuss the CodeRefinery lesson during our StudyGroup 14th December 2018 but we should definitely plan more of best software practices next year.

**Comment by lexnederbragt** [comment]
Why not ask Sabry to lead the session? He is CodeRefinery person at UiO, Carpentries instructor and listed on the first slide.

**Comment by annefou** [comment]
Definitely. I know they are planning to update this lesson so next year would probably be best. I'll check with him.
